doc/level1.rst

```diff
@@ -812,7 +812,10 @@ can generate an Autocrypt Setup Message, and send it to itself.  A
 not-yet-configured Autocrypt MUA (a new MUA in a multi-device
 case, or recovering from device failure or loss) can import the
 Autocrypt Setup Message and recover the ability to read existing
-messages.
+messages. If the Autocrypt MUA is already configured (MUA has secret key
+material in ``accounts[addr].secret_key``) then the UI MUST display a
+warning to the user and ask them to confirm that importing the Autocrypt
+Setup Message will overwrite the existing key.
```

Comment:
maybe add "... and the user may loose access to all mails encrypted to this key."

Reply:
so there is then no way to import the key but retain the old?  I know we don't want to specify historic key state in the L1 spec but i'd rather give the advise that the MUA should keep the old key around for decryption purposes (not specified how) to avoid unreadable mail.  For Delta it's a little less of an issue because it keeps its internal device state independently from the pgp encryption.  for enigmail/k9 that's not the case i think. 

@Valodim or @pbrunschwig Would you for enigmail or K9 really want to throw away ("overwrite") a key that has been used and send out?

Reply:
for Delta Chat, in fact, I plan to keep the old keys, mainly for being able to decrypt new incoming messages still encrypted with the legacy key (already downloaded messages are stored decrypted).

however, I think if we add an general Autocrypt-advice to keep the legacy keys, we should also offer a plan about how to transfer legacy keys later.

Reply:
I interpret "overwrite" as "overwrite the settings". In other words, I would change the configuration to use the new key. But Enigmail would certainly keep the old key as well.

Reply:
This change seems like it's in an odd place in the spec.  The paragraph we're adding to is about the overall utility of the setup message, and the places where it becomes relevant -- when to generate the setup message, when to look for it, and what can be done when processing it.  That's already a lot!

But this change introduces still more: it introduces some additional UI guidance for setup message processing in a context in which we have not yet contemplated even looking for the setup message.

That is, up until this addition, the already-configured Autocrypt MUA apparently has no reason to look at the setup mesage.  Now, we're both giving them a reason, and including some minor guidance of what to do.  

I'd expect this sort of guidance to show up in the `Setup Message Import` section instead, but even that section doesn't yet contemplate the already-configured use case.